flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/table/AggregateITCase.scala

```diff
@@ -114,6 +114,34 @@ class AggregateITCase(mode: StateBackendMode) extends StreamingWithStateTestBase
     assertEquals(expected.sorted, sink.getRetractResults.sorted)
   }
 
+  @Test
+  def testFirstLastValue(): Unit = {
```

Comment:
Could we integrate it into existing test case, e.g. testGroupAggregateWithExpression, etc. Then we could avoid introducing another it case.